configs/nosite-x86_64/variables.yaml

```diff
@@ -4,9 +4,10 @@
 # SPDX-License-Identifier: Apache-2.0
 
 variables:
-  batch_time: ''
-  mpi_command: 'mpirun -n {n_nodes} -c {n_ranks} --oversubscribe'
-  batch_submit: '{execute_experiment}'
-  batch_nodes: ''
-  batch_ranks: ''
-  batch_timeout: ''
+  mpi_command: 'mpirun -n {n_ranks} --oversubscribe'
```

Comment:
Launcher command (mpirun, flux run, srun) always takes n_nodes and n_ranks.

Reply:
Sorry I should have set this to "placeholder", because it is now set by the modifier. But regarding the ability to control nodes/processes in mpirun:

I don't think `mpirun` has an option to request nodes (the way you can provide `-N` to srun) - see: https://www.open-mpi.org/doc/current/man1/mpirun.1.php 

* For example `-N` in mpirun specifies the number of processes to launch per node
* Originally this was 'mpirun -n {n_nodes} -c {n_ranks} --oversubscribe', but from those docs `-c` and `-n` are synonyms

The only mechanism I see for controlling which/how-many nodes is by creating a host file